server/viame_tasks/tasks.py

```diff
@@ -162,9 +153,17 @@ def convert_video(self, path, folderId, token, auxiliaryFolderId):
     stdout, stderr = process.communicate()
     output = str(stdout) + "\n" + str(stderr)
     self.job_manager.write(output)
-    _file = self.girder_client.uploadFileToFolder(folderId, output_path)
+    new_file = self.girder_client.uploadFileToFolder(folderId, output_path)
     self.girder_client.addMetadataToItem(
-        _file["itemId"], {"folderId": folderId, "codec": "h264"}
+        new_file['itemId'], {"folderId": folderId, "codec": "h264",},
```

Comment:
I know your PR doesn't change this, but why are we setting the `meta.folderId` field? This should already be represented in the top level `folderId` field.

Reply:
I still don't get the point of `meta.folderId`.  What items should have this metadata?  What is it used for?

If you can provide a succinct explanation, I can remove it here and add documentation in `README.md`.

Reply:
I also wouldn't mind having better variable names for all the metadata fields.

[`detections, `viame`,`folderId`] are all pretty vague.

Reply:
I don't know what it's used for, I just looked and I don't see it used anywhere else. There used to be a `meta.folderId` on annotation items, but I changed that to `meta.detection` for this exact reason, and it doesn't look like this is related to that at all. I think we're good to remove it.